src/pages/UserHistorySearch.vue

```diff
@@ -66,13 +66,10 @@ export default {
     },
     createFiltersFromURI(uri) {
       const urlSearchParams = new URLSearchParams(uri.split('?').slice(1).pop())
-      const params = Object.fromEntries(urlSearchParams.entries())
-      // Reduce params list into an array
-      return Object.keys(params).reduce((filters, name) => {
-        let value = params[name]
-        // Skip ignored param
+      const filters = []
+      for (let [name, value] of urlSearchParams.entries()) {
```

Comment:
For the sake of simplicity, I would have used a `reduce`

Reply:
we tried to but we could not use a `reduce` on `urlSearchParams.entries()` as it returns an iterator but not an array.
We had extensive discussions about readability of this method and were not completely satisfied, but this is a version we both accepted to yield too :relieved: if you want to try another version we can rediscuss it!